### PR TITLE
refine log

### PR DESCRIPTION
**Pull request by SeaRise**
### What problem does this PR solve?

Issue Number: ref #6233

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Flash/Executor/PipelineExecutor.cpp [hunk 1]
```diff
@@ -108,7 +108,7 @@ ExecutionResult PipelineExecutor::execute(ResultHandler && result_handler)
         scheduleEvents();
         wait();
     }
-    LOG_TRACE(log, "query finish with {}", exec_context.getQueryProfileInfo().toJson());
+    LOG_DEBUG(log, "query finish with {}", exec_context.getQueryProfileInfo().toJson());
```
Comment (SeaRise) [inline dbms/src/Flash/Executor/PipelineExecutor.cpp:111]: may be info level? @windtalker 
```suggestion
    LOG_INFO(log, "query finish with {}", exec_context.getQueryProfileInfo().toJson());
```

Comment (windtalker) [reply dbms/src/Flash/Executor/PipelineExecutor.cpp:111]: LGTM

**Review by windtalker — APPROVED** [review]
LGTM

**Review by Lloyd-Pottiger — APPROVED** [review]

**Pull request merged**